routes/api/v1/recipes.js

```diff
@@ -3,6 +3,24 @@ var router = express.Router()
 var recipes = require('../../../models').Recipe
 var EdamamService = require('../../../services/edamam_service').EdamamService
 
+router.get('/', function(req, res, next) {
+  res.setHeader("Content-Type", "application/json");
+  var edamam = new EdamamService(req.query.food_type)
```

Comment:
@GlynnisOC My understanding for using getRecipes() was just for adding to our database of recipes via the POST request. The recipes index would show all the recipes in our system, not make another request. I suppose we could remove the POST route and use this instead for both populating and showing recipes? We could add a column for food_type to our recipes table so that we just have to do a findAll( {where: { food_type = req.query.food_type }})

Reply:
Thank you for catching!! Index is all set to return all recipes in database, with each recipe's attributes being returned, except for the createdAt and updatedAt attributes.